widgetastic_manageiq/__init__.py

```diff
@@ -446,7 +446,7 @@ def sort_order(self):
         # for filtering the None out
         try:
             return filter(
-                None, re.search(r'sorting_(asc|desc)|fa-sort-(asc|desc)', klass).groups())[0]
+                None, re.search(r'(sorting_|fa-sort-)(asc|desc)', klass).groups())[0]
```

Comment:
I think with the change to the matching groups, we need to return the 1st element of the groups list, not the 0th.

@jawatts Is also making updates in this area and we'll resolve it there if it breaks sorting - containers are the only ones using table sort at the moment.

Reply:
@mshriver correct observation, we should fix that asap - @jawatts please take care